fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/logging/LogMessageKeys.java

```diff
@@ -52,6 +52,7 @@
     MAX_ATTEMPTS("max_attempts"),
     DELAY("delay"),
     COMMIT_NAME("commit_name"),
+    UUID("uuid"),
```

Comment:
Is this still used as a `LogMessageKey` somewhere?

Reply:
No. Removed.